src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```diff
@@ -12,13 +13,19 @@ public class CategoryServiceImpl implements CategoryService {
     private final CategoryRepository categoryRepository;
 
     @Override
-    public Category save(Category category) {
+    public Category create(Category category) {
         return categoryRepository.save(category);
     }
 
+    @Override
+    public Category update(Category category) {
+        return categoryRepository.saveAndFlush(category);
```

Comment:
why flush? I was talking about using `repo.findById().orElseThrow()` to validate if such category even exists first